src/components/AllTags/EditTags.vue

```diff
@@ -39,8 +39,42 @@
           <GoHome />
         </div>
         <TagsSelected />
+        <!-- modal to ask users if they want to add subject agnostic records --->
```

Comment:
Please see if this can made another component and added here like other components

Reply:
Are you aware of any means to modify parent data from the child? I don't think $emit will work for this use case, and that appears to be the only thing I can find. The relevant part is at line 229.

Reply:
I have created another PR with new component

https://github.com/FAIRsharing/FAIRsharing-Assistant/pull/61